packages/components/src/ValidateLabelInput.tsx

```diff
@@ -65,6 +65,7 @@ export function ValidateLabelInput(
       </label>
       {React.Children.toArray(children).map(child => {
         if (!React.isValidElement(child)) {
+          // eslint-disable-next-line react/jsx-key
```

Comment:
We probably should give this a key here and in the `.cloneElement`... though the only key I'd give would just be the index of the array.